lib/project/pro_motion/activities/pm_activity.rb

```diff
@@ -71,6 +71,13 @@ def onOptionsItemSelected(item)
       super
     end
 
+    def onKeyDown(key_code, event)
+      if self.fragment
+        # share onKeyDown with super?
+        super if self.fragment.on_key_down(key_code, event)
```

Comment:
We don't want to always run super, because most of the time back button will leave the app.